workflow_creation/utilities_auto_gui.md

```diff
@@ -1,96 +1,81 @@
-4. **6 decimals** - 6 fields havaing integer or floating point based on default values
-
-The web application is responsible for creation and validation of the fields. It also checks if the field is mandatory, minimum and maximum allowed values.
-
-## Running the Application
-
-The following steps are for using the the Generic Web Application interface for developing the CAxMan applications:
-
-1. Fork or copy the module *Web_Application_Generic*.
-2. Install the **Prerequisites**.
-3. Check if the dependencies are present in the parent path *"/PythonTools"*.
-4. Setting Up the *serverConfiguration* for the server.
-5. Setting Up the *serviceConfiguration* for the Application. This includes providing the path to the application module in the **script** tag and input file if any in **file** tag.
-6. Finally Launch the application service using the followingn command:
-```python
-python webAppInterface.py
-```
-7. The application when accessed in the CAxMan portal should display the generated GUI.
-8. On submission it passes the parameters to the workflow manager. Parameter by parameter. Consequently, you can connect different parameters with different services. Submission **is only allowed, if** all parameters are in their allowed range.
-
+Parameter Extraction for CloudiFacturing Generic User Input asynchronous Web Application
+===
+
+Web applications used for printing typically rely on end user input. In order to prevent malicious parameter inputs, workflow providers have to specify valid parameter ranges for inputs. A suitable way to allow for parameter configuration are spreadsheet files. For this purpose, this repository provides the _GenericGUIApplication_ GUI web application in _code\_examples/Python/app\_generic\_gui_. It allows for interactive parameter specification. When the application runs, it initially constructs appropriate GUI elements for the specified parameters. These GUI elements are incorporated into a web page, which will be presented to the user. The remainder of this document describes the requirements, configuration and deployment of the application.
+
+1 Requirements
+---
+
+You only require to have docker installed. The docker container installs all required packages automatically and runs the application.
+
+2 Configuration
+---
+In order to configure the web application, three different configuration files are used. The following sections describe each of them.
+
+#### 2.1 __Environment Variable File for Service Deployment Parameters:__ 
+The _env_ file from the directory containing this README file specifies environment variables for the docker container. When the application runs, it reads these environment variables. The _env_ file contains all parameters for service deployment and also specifies the used configuration files for web site construction. The listing below shows a complete _env_ file for service deployment configuration.

```

Comment:
"from the directory containing this README file" doesn't make sense, since this markdown document is not in the same place as the env file.

Reply:
I replaced "from the directory containing this README file" with "the app directory" and incorporated a link to the specific directory.